### ISSUE TITLE
[BUG] computed-property-getters throws error with decorators

### ISSUE THREAD

**Issue by Exelord** [ember-cli/eslint-plugin-ember#490]
`computed-property-getters` rule is throwing an error when used with decorators:
`Do not use a getter inside computed properties`
 
```js
  @computed('user.role')
  get myProp() {
    return this.user.role
  }
```

I believe this shouldn't be a case as this is the way recommended by octane guides.

**Comment by bmish** [comment]
CC: @jrjohnson who authored the rule in #397

**Comment by jrjohnson** [comment]
I'm not up to speed on decorators at all, but please assign to me though and I will take a look at this and figure out how to recognize and allow this case.

**Comment by jrjohnson** [comment]
I'm going to need some help getting started with this one. Even trying to create a test I'm seeing 

`Parsing error: Unexpected character \'@\`

Which I'm guessing means we need to add a babel plugin or an option somewhere to support this syntax. I just don't know where to start with that and I'm concerned that we don't really want to lint the transpiled syntax, we want to lint the original as `get` is only allowed inside a decorated computed.

**Comment by Exelord** [comment]
I just found out that the solution can be simpler. We don't need to test it against decorators explicitly. We just need to allow on `computed('property')` with no function or object passed. :) 

I hope that can help

**Comment by Exelord** [comment]
Hmm, that's interesting. I was mentioning it as this doesn't work for me as well :/ 
![image](https://user-images.githubusercontent.com/1469375/63916944-bfb88b80-ca39-11e9-827c-ccd648473105.png)
